pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -47,6 +47,10 @@ const (
 	OVNKubeOverlayNetwork = "OVNKubernetesHybridOverlayNetwork"
 	// kubeProxyServiceName is the name of the kube-proxy Windows service
 	kubeProxyServiceName = "kube-proxy"
+	// kubeletServiceName is the name of the kubelet Windows service
+	kubeletServiceName = "kubelet"
+	// hybridOverlayServiceName is the name of the hybrid-overlay-node Windows service
+	hybridOverlayServiceName = "hybrid-overlay-node"
```

Comment:
The hybrid-verlay related items were in one spot. Any reason to move it?

Reply:
moved it to the top